uber/templates/regform.html

```diff
@@ -116,7 +116,15 @@
             {% else %}
                 setVisible('.badge-row', $.val('amount_extra') >= {{ c.SUPPORTER_LEVEL }});
             {% endif %}
-        };
+            $.each($("input:radio[name='amount_extra']"), function() {
+                $(this).parents('.btn').removeClass('active btn-primary');
+                $(this).parents('.btn').addClass('btn-default');
```

Comment:
``$(this).val()`` is going to return a string, but ``$.val('amount_extra')`` is going to return an integer.  This will happen to work in most situations but will fail in others.  Javascript will automatically do type coercion here but that can be buggy, so it would be better to make sure that you're comparing integers to other integers.